packages/admin-ui/src/filters/MultipleItemsLabel.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { tag } from '@vtex/admin-ui-react'
+
+export const MultipleItemsLabel = (props: { appliedItems: any[] }) => {
+  const { appliedItems } = props
+
+  const firstSelectedItemLabel =
+    appliedItems.length > 1
+      ? `${appliedItems[0]?.label},`
+      : `${appliedItems[0]?.label}` // no comma in this case
+
+  const remainingSelectedItemsCount =
+    appliedItems.length > 1 && `+${appliedItems.length - 1}`
+
+  const appliedValuesLabel = appliedItems.length ? (
+    <>
+      <span>:</span>
+      <tag.span csx={{ color: '$primary' }}>
+        <tag.span
```

Comment:
It would be nice if it was possible to reuse the `SigleItemLabel` here, basically both have the same style and structure. Do you think we ca do it?